xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -95,6 +95,8 @@ void CRenderer::Flush()
     Release(buffer);
 
   ReleaseCache();
+
+  m_subtitlePosResInfo = -1;
```

Comment:
This is pretty much a no-brainer and the implementation is fine as is but wouldn't it be better to wrap this in some `Reset()` class method. What is expected to happen to the other variables that are initialized on class construction like `m_subtitleHorizontalAlign`, `m_subtitlePosition`, `m_subtitleVerticalMargin` etc? I'd expect you want to reset those to the default values too on flush

Reply:
other variables has not so much importance because are resetted in any case
but for clarity i can do as you say i will create a Reset() method